lib/graphql/client.ts

```diff
@@ -37,4 +60,109 @@ export class GraphQLClient {
       throw new GraphQLError(serializeGraphQLError(error));
     }
   };
+
+  /**
+   * Return subscription client connection params
+   */
+  private getConnectionParams = (): ConnectionParams => ({
+    Authorization: `Bearer ${this.auth.tokenManager.token?.accessToken}`
+  });
```

Comment:
Inlining this could even simplify the code a bit, let us discuss that if you want.

Reply:
You are absolutely correct, it was even inlined at first, and I extracted it for helping with other tests or something similar at some point but some other refactorings must have made this extraction pointless.

Great catch 👍
Changed in this commit: 12b9b2e12b288e8c1c0bc88da28e5c03bc422e75